dotCMS/src/main/java/com/dotcms/contenttype/model/field/FieldType.java

```diff
@@ -42,9 +44,14 @@ public String getHelpText() {
     }
 
     public Map<String, Object> toMap(){
+        ImmutableList<String> propertiesName = properties.stream()
+                .map(property -> property.getName())
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Specify a type for: 'property' [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2211)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1612)